Detectors/TPC/simulation/include/TPCSimulation/Point.h

```diff
@@ -175,18 +173,18 @@ class LinkableHitGroup : public o2::BaseHit {
 #ifdef HIT_AOS
   std::vector<o2::TPC::ElementalHit> mHits; // the hits for this group
 #else
-  std::vector<float> mHitsXVctr; //! do not stream this (just for memory handling convenience)
-  std::vector<float> mHitsYVctr; //! do not stream this
-  std::vector<float> mHitsZVctr; //! do not stream this
-  std::vector<float> mHitsTVctr; //! do not stream this
-  std::vector<short> mHitsEVctr; //! do not stream this
+  std::vector<float> mHitsXVctr; // do not stream this (just for memory handling convenience)
+  std::vector<float> mHitsYVctr; // do not stream this
+  std::vector<float> mHitsZVctr; // do not stream this
+  std::vector<float> mHitsTVctr; // do not stream this
+  std::vector<short> mHitsEVctr; // do not stream this
   // let us stream ordinary buffers for compression AND ROOT IO/speed!!
   Int_t mSize;
-  float* mHitsX = nullptr; //[mSize]
-  float* mHitsY = nullptr; //[mSize]
-  float* mHitsZ = nullptr; //[mSize]
-  float* mHitsT = nullptr; //[mSize]
-  short* mHitsE = nullptr; //[mSize]
+  float* mHitsX = nullptr; //![mSize]
+  float* mHitsY = nullptr; //![mSize]
+  float* mHitsZ = nullptr; //![mSize]
+  float* mHitsT = nullptr; //![mSize]
+  short* mHitsE = nullptr; //![mSize]
```

Comment:
If these should not be streamable (//!) then I guess you also don't need the '[mSize]'. See also my comment above (class needs to be streamable or not).